components/global_header/right_controls/right_controls.tsx

```diff
@@ -55,6 +57,7 @@ const RightControls = ({productId = null}: Props): JSX.Element => {
                     pluggableId={productId}
                 />
             )}
+            {productId === null ? '' : <UserGuideDropdown/>}
```

Comment:
```suggestion
            {currentProduct?.pluginId === 'playbooks' && <UserGuideDropdown/>}
```

Reply:
I think this would work to get the current product:
```jsx
const {pathname} = useLocation();
const currentProduct = useSelector((state: GlobalState) => getCurrentProduct(state.plugins.components.Product, pathname));

```